mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -799,6 +799,14 @@ def test_crop():
         assert_equal(raw.last_samp, ar.last_samp)
         assert_array_equal(raw[:, :][0], ar[:, :][0])
 
+    # test shape consistency of cropped raw
+    data = np.zeros((1, 1002001))
+    info = create_info(['1'], 1000)
+    raw = RawArray(data, info)
+    for tmin in np.arange(0, 1001, 100):
+        raw1 = raw.crop(tmin=tmin, tmax=tmin + 2, copy=True)
+        assert_equal(raw1._data.shape, (1, 2001))
```

Comment:
(sometimes it might be worth using private attributes for speed or memory reasons, but I don't think that's the case here...?)